src/main/java/com/bobocode/petros/injector/AnnotationDependencyInjector.java

```diff
@@ -46,25 +46,27 @@ public AnnotationDependencyInjector(String packageName) {
     public Map<DependencyDefinition, Object> injectedDependencyDefinitionObjectMap() {
         var scannedConfigurationsMap = configurationScanner.scan(packageName);
         var scannedClassesMap = classScanner.scan(packageName);
-        createConfigClassDependencies(scannedConfigurationsMap);
-        createConfigClassDependencies(scannedClassesMap);
+        LOG.info("Creating config class dependencies...");
+        createDependencies(scannedConfigurationsMap);
```

Comment:
It will be useful also to LOG info that the creation operation was successful, like
LOG.info("Config class dependencies were successfully created") or smth

Reply:
I wrote it inside the methods of creating